metashare/api/models.py

```diff
@@ -676,7 +678,7 @@ def save(self, *args, **kwargs):
         ret = super().save(*args, **kwargs)
 
         if is_new:
-            self.queue_provision(originating_user_id=None)
+            self.queue_provision(originating_user_id=str(self.owner.id))
```

Comment:
@wlonk Without this, we were losing provision front-end messages. Feel free to adjust as you like.

Reply:
@wlonk Take a look at other cases where we're using `originating_user_id=None`?